### PR TITLE
Aggregate census data to the town level

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/models/grda_warehouse/shape/town.rb [hunk 1]
```diff
@@ -10,12 +10,29 @@ class Town < GrdaWarehouseBase
       include SharedBehaviors
       include StateScopes
 
-      def name
-        read_attribute(:town)
+      def candidate_zip_codes
```
Comment (yakloinsteak) [inline app/models/grda_warehouse/shape/town.rb:13]: I would have used census block groups, but this is fine.

File: app/models/grda_warehouse/us_census_api/town_agg.rb [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# https://www.census.gov/content/dam/Census/data/developers/api-user-guide/api-guide.pdf
+# https://www.census.gov/programs-surveys/geography/guidance/geo-identifiers.html
+
+module GrdaWarehouse
+  module UsCensusApi
+    class TownAgg
```
Comment (yakloinsteak) [inline app/models/grda_warehouse/us_census_api/town_agg.rb:12]: Looks okay to me. Were you just saving time by not updating the CoC one to be more generic?

**Review by yakloinsteak — APPROVED** [review]

File: app/models/grda_warehouse/shape/town.rb (hunk 1 above)
Comment (eanders) [reply app/models/grda_warehouse/shape/town.rb:13]: Yeah, it turned out we don't have census block groups loaded.

File: app/models/grda_warehouse/us_census_api/town_agg.rb (hunk 2 above)
Comment (eanders) [reply app/models/grda_warehouse/us_census_api/town_agg.rb:12]: Didn't even think of it, but yes, time was  a constraint.

**Pull request merged**